Runtimes/Supplemental/Differentiation/CMakeLists.txt

```diff
@@ -8,48 +17,63 @@ project(SwiftDifferentiation
   LANGUAGES Swift C
   VERSION 6.1.0${BUILD_NUMBER})
 
+if(NOT PROJECT_IS_TOP_LEVEL)
+  message(SEND_ERROR "Swift Differentiation must build as a standalone project")
+endif()
+
 set(CMAKE_POSITION_INDEPENDENT_CODE YES)
 set(CMAKE_Swift_LANGUAGE_VERSION 5)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake/modules")
 
-if(NOT PROJECT_IS_TOP_LEVEL)
-  message(SEND_ERROR "Swift Differentiation must build as a standalone project")
-endif()
-
 set(${PROJECT_NAME}_SWIFTC_SOURCE_DIR
   "${PROJECT_SOURCE_DIR}/../../../"
   CACHE FILEPATH "Path to the root source directory of the Swift compiler")
 
-find_package(SwiftCore)
+# Hook point for vendor-specific extensions to the build system
+# Allowed extension points:
+#   - DefaultSettings.cmake
```

Comment:
This comment is not in sync with the content.
It looks like the only vendor module file for this project is `swift_Differentiation.cmake`.